frontend/common/services/useRole.ts

```diff
@@ -6,38 +6,34 @@ export const roleService = service
   .enhanceEndpoints({ addTagTypes: ['Role'] })
   .injectEndpoints({
     endpoints: (builder) => ({
-      createRole: builder.mutation<Res['roles'], Req['createRoles']>({
```

Comment:
there was no definition of Res['roles'], Req['createRoles'] etc in this file, all were typos